client/packages/programs/src/JsonForms/common/components/Array.tsx

```diff
@@ -57,11 +61,50 @@ interface EnumArrayControlCustomProps extends ArrayControlProps {
   data: string[];
 }
 
+const Options = z
+  .object({
+    detail: z.object({ type: z.string(), elements: z.array(z.any()) }),
```

Comment:
Had to make this "any" as there was no way to access existing Type definitions (i.e. "UiSchemaElement"). Should be okay though, as JSONForms itself already validates these elements.